### PR TITLE
add a11y configuration for title

### PR DESCRIPTION
**Pull request by evilvillage**
Add capability to customize title text for a11y.

### CODE REVIEW

File: src/js/mep-player.js [hunk 1]
```diff
@@ -338,6 +338,11 @@ function isAfter(sourceNode, targetNode) {
 				t.$media.removeAttr('controls');
 				var videoPlayerTitle = t.isVideo ?
 					mejs.i18n.t('Video Player') : mejs.i18n.t('Audio Player');
+
+				if (t.options.titleText) {
+					videoPlayerTitle = t.options.titleText;
```
Comment (benternet) [inline src/js/mep-player.js:345]: do we need a spec that the title can be set?

**Review by benternet — CHANGES_REQUESTED** [review]

File: src/js/mep-player.js (hunk 1 above)
Comment (zellv) [reply src/js/mep-player.js:345]: Don't think so.

**Comment by mziwisky** [comment]
@evilvillage please rebase

**Review by mziwisky — CHANGES_REQUESTED** [review]
can you revert all those whitespace changes?

**Review by mziwisky — APPROVED** [review]
+2

**Pull request merged**